cmd/torpedo/torpedo.go

```diff
@@ -667,12 +691,15 @@ func fireTorpedo(c *cli.Context) {
 	} else if nodeDriver, err := node.Get(n); err != nil {
 		logrus.Fatalf("Cannot find node driver for %v. Err: %v\n", n, err)
 		os.Exit(-1)
+	} else if err := os.MkdirAll(logLoc, os.ModeDir); err != nil {
+		logrus.Fatalf("Cannot create path %s for saving support bundle. Error was: %v", logLoc, err)
```

Comment:
nit: Instead of "Error was" just say "Err:".

Reply:
Done